library/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.jboss.aerogear.android.core">
+
+    <application
+        android:allowBackup="true"
+        android:label="@string/app_name"
+        android:supportsRtl="true">
+    </application>
```

Comment:
The application node causes problems in downstream projects.  We don't really want to push these values down since we are a library and should probably remove them.